src/WebJobs.Script/Utility.cs

```diff
@@ -792,6 +792,35 @@ public static bool IsMediaTypeOctetOrMultipart(MediaTypeHeaderValue mediaType)
             }
         }
 
+        public static string ReadAutorestGeneratedJson(string rootScriptPath)
```

Comment:
Pass in the logger as input - move the logic to log into this method. Rename this  method: `LogAutorestGeneratedJson`

Reply:
Moving the logic, will allow you to write unit tests - please add unit tests in `UtilityTests`